.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# workflow name
+name: Generate release-artifacts
+
+# on events
+on:
+  push:
+    tags:
+        - '*'
+
+# workflow tasks
+jobs:
+  generate:
+    name: Generate cross-platform builds
+    runs-on: ubuntu-latest
+    steps:
+      - uses: olegtarasov/get-tag@v2.1
+        id: tagName
+        with:
+          tagRegex: "v(.*)"  # Optional. Returns specified group text as tag name. Full tag string is returned if regex is not defined.
+          tagRegexGroup: 1 # Optional. Default is 1.
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Generate build files
+        uses: thatisuday/go-cross-build@v1.1.0
+        with:
+            platforms: 'linux/amd64'
+            package: 'src'
+            name: 'foreman_ygg_worker-${{ steps.tagName.outputs.tag }}'
```

Comment:
I believe GitHub has tag in metadata somewhere.
Would be nice to limit the number of action dependencies.
I'd be open to tagging without the 'v' prefix if that would help

Reply:
It has, but it looks like `refs/tags/v0.0.1`. For the time being I just pulled this dependency. If we decide this is the way we want to go about it, then I'll probably try extracting the tag from the metadata instead of having a 3rd party dependency do so.